### ISSUE TITLE
please provide more info

### ISSUE THREAD

**Issue by andrzejsliwa** [procore/blueprinter#52]
Dear @dlcarter,

This gem, looks like new version of your ApplicationSerializer mentioned by you in "RailsConf 2017: ​Rails APIs: The Next Generation" talk. I'm right about it? (semantic looks almost identical). Can you provide some examples about how to integrate it with Rails application? Do you have any new remarks from using it in your application? Maybe some blog post about it?

**Comment by philipqnguyen** [comment]
Hello @andrewcrockett 

@dlcarter is away due to the holidays, so I will try to help you, and he can chime in after. While Blueprinter is currently open source, we haven't released it with a 0.1.0 version yet. We don't have a blog post on it yet, but you should hear more about it once we make 0.1.0, which should not be long. You can still give it a try, assuming inside a rails app:

```ruby
# Gemfile
gem 'blueprinter', github: 'procore/blueprinter', branch: 'master'
```

```ruby
# app/blueprints/user_blueprint
class UserBlueprint < Blueprinter::Base
  identifier :uuid # or you can leave this out to default to :id
  field :email, name: :login

  view :normal do
    fields :first_name, :last_name
  end

  view :extended do
    include_view :normal
    field :address
  end
end
```

```ruby
# app/users_controllers
class UsersController < ApplicationController
  def index
    render json: UserBlueprint.render(User.all, view: :normal)
  end

  def show
    user = User.find params[:id]
    render json: UserBlueprint.render(user, view: :extended)
  end
end
```

**Comment by philipqnguyen** [comment]
We also welcome any help with issues or PRs as well!

**Comment by philipqnguyen** [comment]
@andrzejsliwa the gem is released, currently 0.2.0. So a typical `gem install blueprinter` would work now. I'll close this for now, but if you have any other questions, feel free to re-open or start a new issue.
